gnd/src/main/java/com/google/android/gnd/ui/common/ViewModelModule.java

```diff
@@ -96,6 +97,11 @@
   @ViewModelKey(EditObservationViewModel.class)
   abstract ViewModel bindEditObservationViewModel(EditObservationViewModel viewModel);
 
+  @Binds
+  @IntoMap
+  @ViewModelKey(PhotoFieldViewModel.class)
+  abstract ViewModel bindPhotoViewViewModel(PhotoFieldViewModel viewModel);
```

Comment:
Naming nit: Our classes follow the pattern FooView and FooViewModel, which would make this class here "PhotoViewModel". To make it clear this is specifically a photo in a form field, PhotoFieldView and PhotoFieldViewModel might be more appropriate.